### PR TITLE
added two search bars

### PR DESCRIPTION
**Pull request by vsupruniuk**
- [DEMO LINK](https://vsupruniuk.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://vsupruniuk.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,44 @@
...
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://www.airbnb.com.ua/"
+      method="get"
+      data-qa="big"
+      class="form"
+    >
+
+      <label class="search__big">
```
Comment (Olexandr6) [inline src/index.html:20]:
```suggestion
    >
      <label class="search__big">
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,44 @@
...
+
+        <input type="text"
+          data-qa="keypress"
+          class="search__bar search__bar1"
+          placeholder="Try &quot;Los Angeles&quot;"
+        >
+
+      </label>
+    </form>
+    <form action="https://www.airbnb.com.ua/"
```
Comment (Olexandr6) [inline src/index.html:34]:
```suggestion
    </form>
    
    <form action="https://www.airbnb.com.ua/"
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,44 @@
...
+    method="get"
+    data-qa="small"
+    class="form"
```
Comment (Olexandr6) [inline src/index.html:37]: Check the 4th point from the [cheklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

**Review by Olexandr6 — CHANGES_REQUESTED** [review]

**Review by jstmpelowycz — APPROVED** [review]
You're done, congrats😎

**Review by illiarerun — APPROVED** [review]

**Review by jstmpelowycz — APPROVED** [review]
Re-approve

**Review by Olexandr6 — APPROVED** [review]

**Pull request closed without merging**